pysumma/Decisions.py

```diff
@@ -15,6 +15,7 @@ def __init__(self, filepath):
         self.LAI_method = DecisionOption(self, 'LAI_method')
         self.f_Richards = DecisionOption(self, 'f_Richards')
         self.groundwatr = DecisionOption(self, 'groundwatr')
+        self.rootProfil = DecisionOption(self, 'rootProfil')
```

Comment:
I think we have to edit SUMMA Test Case before merging this commit. Do you have updated Test Cases and SUMMA version?